subgraphs/uniswap-forks/protocols/trader-joe/src/common/handlers/handleRewardV2.ts

```diff
@@ -46,11 +47,13 @@ export function updateMasterChef(
 
   // Calculate Reward Emission per second to a specific pool
   // Pools are allocated based on their fraction of the total allocation times the rewards emitted per second
-  const rewardAmountPerInterval = masterChefV2.adjustedRewardTokenRate
-    .times(masterChefV2Pool.poolAllocPoint)
-    .div(masterChefV2.totalAllocPoint);
-  const rewardAmountPerIntervalBigDecimal = BigDecimal.fromString(
-    rewardAmountPerInterval.toString()
+  const rewardAmountPerIntervalBigDecimal = safeDiv(
+    new BigDecimal(
+      masterChefV2.adjustedRewardTokenRate.times(
+        masterChefV2Pool.poolAllocPoint
+      )
+    ),
+    new BigDecimal(masterChefV2.totalAllocPoint)
```

Comment:
nit: is a 0 value possible here or is there another issue? if it can happen then we are good, otherwise we should log the error where it happens.

If not, feel free to resolve and merge

Reply:
the value for `masterChefV2.totalAllocPoint` is actually 0 now: https://snowtrace.io/address/0xd6a4F121CA35509aF06A0Be99093d08462f53052#readContract#F13